vnext/src/Libraries/Pressability/Pressability.windows.js

```diff
@@ -562,6 +578,13 @@ export default class Pressability {
         ? null
         : {
             onMouseEnter: (event: MouseEvent): void => {
+              // [Windows Prefer attached raw mouse event handler for compat
+              if (this._config.onMouseEnter) {
+                this._config.onMouseEnter(event);
+                return;
```

Comment:
Lets remove the return here, (and on mouseLeave), so that we dont break the hover logic